HW_8/Task1.py

```diff
@@ -0,0 +1,11 @@
+#Напишите функцию, которая преобразует pickle файл хранящий список словарей в табличный csv файл. 
+#Для тестированию возьмите pickle версию файла из предыдущей задачи. Функция должна извлекать ключи словаря 
+#для заголовков столбца из переданного файла.
+
+import pickle
+import pandas
+
+with open("Task5.pickle", "rb") as f:
+    object1 = pickle.load(f)
+df = pandas.DataFrame(object1)
+df.to_csv(r'file.csv')
```

Comment:
Хитро, но вы же все равно объект DataFrame не используете.
```suggestion
pandas.DataFrame(object1).to_csv(r'file.csv')
```